relentless/optimize/method.py

```diff
@@ -339,14 +430,21 @@ def optimize(self, objective):
         iter_num = 0
         cur_res = objective.compute()
         while not self.has_converged(cur_res) and iter_num < self.max_iter:
+            grad_hat = cur_res.gradient*self.grad_scale**2
+            c = self.descent_amount(grad_hat)
+            update = c*grad_hat/grad_hat.norm()
```

Comment:
Along with my comment above, I'm in favor of not normalizing the gradient here, and making the `descent_amount` method decide to do it.